### ISSUE TITLE
Query on BODY_PARTS_PAF_IDS 

### ISSUE THREAD

**Issue by nishchaljs** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#92]
I wanted to train a Custom model for detecting FOOT Keypoints. I got an idea of how to configure BODY_PARTS_KPT_IDS. But i am not so sure how to modify the BODY_PARTS_PAF_IDS. Can you please help me in the same?
Left Ankle-0
Left heel-1
Left big toe-2
Left small toe-3
Right Ankle-4
Right heel-5
Right big toe-6
Right small toe-7

BODY_PARTS_KPT_IDS=[[0, 1], [1, 2], [1, 3], [4, 5], [5, 6], [5, 7]]
BODY_PARTS_PAF_IDS =????????

**Comment by Daniil-Osokin** [comment]
Hi! Mind to check [TRAIN-ON_CUSTOM_DATASET.md](https://github.com/Daniil-Osokin/lightweight-human-pose-estimation.pytorch/blob/master/TRAIN-ON-CUSTOM-DATASET.md)?

**Comment by nishchaljs** [comment]
Hi, I am training it on Foot Dataset which is in coco format now, the losses seems to stagnate after 20000 iterations, I just followed the instructions on Train-On-Custom_Dataset.md file you attached.. Am i missing something?

iter: 30000
stage1_pafs_loss:     85.11190738677979
stage1_heatmaps_loss: 33.45372098922729
stage2_pafs_loss:     72.09435405731202
stage2_heatmaps_loss: 27.586418838500975

One more request, is it possible for you to upload codes for any custom dataset like Foot or Hand? It would be really helpful, thanks!!

**Comment by Daniil-Osokin** [comment]
If loss does not decrease for a while, possibly you need to drop learning rate. Learning rate drop schedule is specific for the dataset. There is code only for human pose estimation, I do not have other codes for this project.

**Comment by nishchaljs** [comment]
When I test the model I have trained, I noticed that there is a lot of confusion between the predictions of left and right foot..
The keypoints responsible for right foot sometimes detect the left side as right foot and vice-versa..
I am not sure why this is happening, any idea??

**Comment by Daniil-Osokin** [comment]
Hi! When you add a new dataset to the old pipeline or test new pipeline with the well-known dataset there is a great rule: try to overfit on small number of images, e.g. take 2-16 images and train on them. If everything is ok, the you should have loss about 0 and perfect results on these images. If not, then something in new dataset/pipeline needs to be fixed. Did you try something like this? In general, it should work ok, since there are similar models for foot pose estimation, so suggest to overfit (debug) on small  subset of images and check what went wrong.

**Comment by Daniil-Osokin** [comment]
Hope, it helped.
